test/e2e/density.go

```diff
@@ -632,6 +661,48 @@ var _ = framework.KubeDescribe("Density", func() {
 			}
 		})
 	}
+
+	// Calculate total number of pods from each node's max-pod
+	It("[Feature:ManualPerformance] should allow running maximum capacity pods on nodes", func() {
+		totalPods = 0
+		for _, n := range nodes.Items {
+			totalPods += int(n.Status.Capacity.Pods().Value())
+		}
+		totalPods -= framework.WaitForStableCluster(c, masters)
+
+		fileHndl, err := os.Create(fmt.Sprintf(framework.TestContext.OutputDir+"/%s/pod_states.csv", uuid))
```

Comment:
The disk usage should end up being similar to other density tests and they haven been small text files that haven't caused a problem (yet).  This data file is pretty important for our performance evaluations, so for the time being I think I would like to keep it like the other density tests.  If this file causes disk space issues something is seriously wrong (like the test timeout doesn't work).